src/components/core/wizard/wizard.tsx

```diff
@@ -0,0 +1,401 @@
+import CloseIconDark from '@assets/svg/modal/ic-close-dark.svg';
+import CloseIconLight from '@assets/svg/modal/ic-close-light.svg';
+import BackArrowIconLight from '@assets/svg/wizard/ic-back-arrow-light.svg';
+import BackArrowIconDark from '@assets/svg/wizard/ic-back-arrow-dark.svg';
+import Button from '@core/button/button';
+import Text from '@core/text/text';
+import React from 'react';
+import { styled } from 'Styles/stitches.config';
+import StepNavigation from './step-navigation';
+
+const DarkBackgroundContainer = styled('div', {
+    position: 'absolute',
+    inset: '0',
+    background: 'rgba(0, 0, 0, 0.72)',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+
+    variants: {
+        visible: {
+            false: {
+                display: 'unset',
+                background: 'unset',
+            },
```

Comment:
[Clarification] So this is only in place when we pass `visible={false}`?

Reply:
@sean-binary yes🙂